src/EspressoNitroTEEVerifier.sol

```diff
@@ -1,96 +1,70 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {NitroValidator} from "@nitro-validator/NitroValidator.sol";
-import {LibBytes} from "@nitro-validator/LibBytes.sol";
-import {LibCborElement, CborElement, CborDecode} from "@nitro-validator/CborDecode.sol";
-import {CertManager} from "@nitro-validator/CertManager.sol";
 import "@openzeppelin/contracts/access/Ownable2Step.sol";
 import {IEspressoNitroTEEVerifier} from "./interface/IEspressoNitroTEEVerifier.sol";
+import {
+    INitroEnclaveVerifier,
+    VerifierJournal,
+    ZkCoProcessorType
+} from "aws-nitro-enclave-attestation/interfaces/INitroEnclaveVerifier.sol";
 
 /**
  * @title  Verifies quotes from the AWS Nitro Enclave (TEE) and attests on-chain
- * @notice Contains the logic to verify an attestation and signature from the TEE and attest on-chain. It uses the NitroValidator contract
- *         from `base` to verify the quote. Along with some additional verification logic.
- *         (https://github.com/base/nitro-validator)
- * The code of this contract is inspired from SystemConfigGlobal.sol
- * (https://github.com/base/op-enclave/blob/main/contracts/src/SystemConfigGlobal.sol)
+ * @notice Contains the logic to verify zk proof of the attestation on-chain. It uses the EspressoNitroTEEVerifier contract
+ *         from `automata` to verify the proof.
  */
-contract EspressoNitroTEEVerifier is NitroValidator, IEspressoNitroTEEVerifier, Ownable2Step {
-    using CborDecode for bytes;
-    using LibBytes for bytes;
-    using LibCborElement for CborElement;
-
+contract EspressoNitroTEEVerifier is IEspressoNitroTEEVerifier, Ownable2Step {
     // PCR0 keccak hash
     mapping(bytes32 => bool) public registeredEnclaveHash;
     // Registered signers
     mapping(address => bool) public registeredSigners;
-    // Certificate Manager
-    CertManager _certManager;
+    // Nitro Enclave Verifier
+    INitroEnclaveVerifier public _nitroEnclaveVerifier;
 
-    constructor(bytes32 enclaveHash, CertManager certManager)
-        NitroValidator(certManager)
-        Ownable()
-    {
-        _certManager = certManager;
+    constructor(bytes32 enclaveHash, INitroEnclaveVerifier nitroEnclaveVerifier) Ownable2Step() {
+        _nitroEnclaveVerifier = nitroEnclaveVerifier;
         registeredEnclaveHash[enclaveHash] = true;
         _transferOwnership(msg.sender);
     }
 
     /**
      * @notice This function registers a new signer by verifying an attestation from the AWS Nitro Enclave (TEE)
-     * @param attestation The attestation from the AWS Nitro Enclave (TEE)
-     * @param signature The cryptographic signature over the COSESign1 payload (extracted from the attestation)
+     * @param output The output from the AWS Nitro Enclave (TEE)
```

Comment:
I think the comment has to be updated to specify that it now verifies a zero-knowledge proof of AWS Nitro Enclave attestation